components/shell/bash/Makefile

```diff
@@ -107,20 +114,16 @@ CONFIGURE_OPTIONS  += 		--enable-restricted
 CONFIGURE_OPTIONS  += 		--enable-select
 CONFIGURE_OPTIONS  += 		--enable-separate-helpfiles
 CONFIGURE_OPTIONS  += 		--enable-single-help-strings
-CONFIGURE_OPTIONS  += 		--disable-strict-posix-default
+CONFIGURE_OPTIONS  +=			--enable-static-link
 CONFIGURE_OPTIONS  += 		--enable-usg-echo-default=yes
 CONFIGURE_OPTIONS  += 		--enable-xpg-echo-default=yes
-CONFIGURE_OPTIONS  += 		--enable-mem-scramble
-CONFIGURE_OPTIONS  += 		--disable-profiling
-CONFIGURE_OPTIONS  += 		--enable-static-link
-CONFIGURE_OPTIONS  += 		--enable-largefile
-CONFIGURE_OPTIONS  += 		--enable-nls
+CONFIGURE_OPTIONS  +=			--infodir=$(CONFIGURE_INFODIR)
+CONFIGURE_OPTIONS  +=			--localstatedir=/var
 CONFIGURE_OPTIONS  += 		--with-bash-malloc=yes
 CONFIGURE_OPTIONS  += 		--with-curses
 CONFIGURE_OPTIONS  += 		--with-installed-readline=no
-CONFIGURE_OPTIONS  +=		--infodir=$(CONFIGURE_INFODIR)
 
-COMPONENT_TEST_DIR =		$(BUILD_DIR_32)
+COMPONENT_TEST_DIR =		$(BUILD_DIR_64)
```

Comment:
Just that autoreconf fails since all the AC_DEFINE occurences should be fixed and I had no motivation to spend time on this...